test/Enum.rb

```diff
@@ -1,226 +1,278 @@
-    enum3 = Magick::Enum.new(:foo, 36)
-    enum4 = Magick::Enum.new(:foo, 42)
-
-    expect(enum1 <=> enum2).to eq(-1)
-    expect(enum1 <=> enum4).to eq(0)
-    expect(enum1 <=> enum3).to eq(1)
-    expect(enum1 <=> 'x').to be(nil)
+  describe '#spaceship' do
+    it 'works' do
+      enum1 = Magick::Enum.new(:foo, 42)
+      enum2 = Magick::Enum.new(:foo, 56)
+      enum3 = Magick::Enum.new(:foo, 36)
+      enum4 = Magick::Enum.new(:foo, 42)
+
+      expect(enum1 <=> enum2).to eq(-1)
+      expect(enum1 <=> enum4).to eq(0)
+      expect(enum1 <=> enum3).to eq(1)
+      expect(enum1 <=> 'x').to be(nil)
+    end
   end
 
-  def test_case_eq
-    enum1 = Magick::Enum.new(:foo, 42)
-    enum2 = Magick::Enum.new(:foo, 56)
+  describe '#case_eq' do
+    it 'works' do
+      enum1 = Magick::Enum.new(:foo, 42)
+      enum2 = Magick::Enum.new(:foo, 56)
 
-    expect(enum1 === enum1).to be(true)
-    expect(enum1 === enum2).to be(false)
-    expect(enum1 === 'x').to be(false)
+      expect(enum1 === enum1).to be(true)
+      expect(enum1 === enum2).to be(false)
+      expect(enum1 === 'x').to be(false)
+    end
   end
 
-  def test_bitwise_or
-    enum1 = Magick::Enum.new(:foo, 42)
-    enum2 = Magick::Enum.new(:bar, 56)
+  describe '#bitwise_or' do
+    it 'works' do
+      enum1 = Magick::Enum.new(:foo, 42)
+      enum2 = Magick::Enum.new(:bar, 56)
 
-    enum = enum1 | enum2
-    expect(enum.to_i).to eq(58)
-    expect(enum.to_s).to eq('foo|bar')
+      enum = enum1 | enum2
+      expect(enum.to_i).to eq(58)
+      expect(enum.to_s).to eq('foo|bar')
 
-    expect { enum1 | 'x' }.to raise_error(ArgumentError)
+      expect { enum1 | 'x' }.to raise_error(ArgumentError)
+    end
   end
 
-  def test_type_values
-    expect(Magick::AlignType.values).to be_instance_of(Array)
+  describe '#type_values' do
+    it 'works' do
+      expect(Magick::AlignType.values).to be_instance_of(Array)
 
-    expect(Magick::AlignType.values[0].to_s).to eq('UndefinedAlign')
-    expect(Magick::AlignType.values[0].to_i).to eq(0)
+      expect(Magick::AlignType.values[0].to_s).to eq('UndefinedAlign')
+      expect(Magick::AlignType.values[0].to_i).to eq(0)
 
-    Magick::AlignType.values do |enum|
-      expect(enum).to be_kind_of(Magick::Enum)
-      expect(enum).to be_instance_of(Magick::AlignType)
+      Magick::AlignType.values do |enum|
+        expect(enum).to be_kind_of(Magick::Enum)
+        expect(enum).to be_instance_of(Magick::AlignType)
+      end
     end
   end
 
-  def test_type_inspect
-    expect(Magick::AlignType.values[0].inspect).to eq('UndefinedAlign=0')
+  describe '#type_inspect' do
+    it 'works' do
+      expect(Magick::AlignType.values[0].inspect).to eq('UndefinedAlign=0')
+    end
   end
 
-  def test_using_compose_does_not_cause_endless_loop
-    img = Magick::Image.new(10, 10)
-    Magick::CompositeOperator.values do |op|
-      img.compose = op
-      expect(img.compose).to eq(op)
+  describe '#using_compose_does_not_cause_endless_loop' do
```

Comment:
Wonder if we could get rid of the underscores. Or do this:
```
describe '#compose' do
  it 'does not cause endless loop`
```
And move it to the other file?

Reply:
Yeah, I figured I'd clean these up when I'm splitting up the spec files a little further along.

Reply:
Okay 👍